util/reggen/reg_pkg.sv.tpl

```diff
@@ -365,8 +367,9 @@ package ${lblock}_reg_pkg;
 % endfor
 <%
   just_default = len(block.reg_blocks) == 1 and None in block.reg_blocks
+  reg_blocks = block.reg_blocks
 %>\
-% for iface_name, rb in block.reg_blocks.items():
+% for iface_name, rb in reg_blocks.items():
```

Comment:
Nit: Not a problem, but any particular reason to define the local variable?

Reply:
jep, let's simplify this.